lib/test.js

```diff
@@ -7,6 +7,16 @@ import concordanceOptions from './concordance-options.js';
 import nowAndTimers from './now-and-timers.cjs';
 import parseTestArgs from './parse-test-args.js';
 
+function isExternalAssertError(error) {
+	// Match errors thrown by <https://www.npmjs.com/package/expect>.
+	if (error.matcherResult) {
+		return true;
+	}
+
+	// Match errors thrown by <https://www.npmjs.com/package/chai> and <https://nodejs.org/api/assert.html>.
+	return error.hasOwnProperty('actual') && error.hasOwnProperty('expected');
```

Comment:
I noticed the properties can have value `false` (which is the case in the test). So we should check existence of the property instead of the property. (which was the case in the original PR)